scripts/jest/root.config.js

```diff
@@ -7,16 +7,5 @@
 module.exports = {
     rootDir: '../..',
     testMatch: ['<rootDir>/**/__tests__/*.spec.(js|ts)'],
-    projects: [
-        '<rootDir>/packages/@lwc/errors',
-        '<rootDir>/packages/@lwc/babel-plugin-component',
-        '<rootDir>/packages/@lwc/compiler',
-        '<rootDir>/packages/@lwc/engine',
-        '<rootDir>/packages/@lwc/module-resolver',
-        '<rootDir>/packages/@lwc/template-compiler',
-        '<rootDir>/packages/@lwc/style-compiler',
-        '<rootDir>/packages/@lwc/wire-service',
-        '<rootDir>/packages/@lwc/rollup-plugin',
-        '<rootDir>/packages/@lwc/synthetic-shadow',
-    ],
+    projects: ['<rootDir>/packages/@lwc/*'],
```

Comment:
`@lwc/features` unit tests weren't running because I forgot to add that to this list.